src/Tribe/Aggregator/Processes/Batch_Imports.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * Class Tribe__Events__Aggregator__Processes__Batch_Imports
+ *
+ * @since TBD
+ */
+
+/**
+ * Class Tribe__Events__Aggregator__Processes__Batch_Imports
+ *
+ * Add custom hooks in order to support batch pushing.
+ *
+ * @since TBD
+ */
+class Tribe__Events__Aggregator__Processes__Batch_Imports {
+	/**
+	 * Attach hooks to support batch pushing.
+	 *
+	 * @since TBD
+	 */
+	public function hook() {
+		add_filter( 'tribe_events_aggregator_build_url', [ $this, 'build_url' ], 10, 3 );
```

Comment:
Move this out of this, and thus remove the need for this to be a singleton and the accompanying test issues, and proxy this hooking into the service provider as done elsewhere.

As a general rule, first-level hooking (as is the case here) should not be done in the class, rather than in the service provider managing it, to decouple the class from its contextual use.